containers/package-builder/build-virtualenv.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+concent_dir=$(realpath "$1")
+build_dir=$(realpath "$2")
+
+mkdir --parents "$build_dir/virtualenv/"
+cp --recursive "$concent_dir/concent_api/requirements.lock" "$build_dir/virtualenv"
```

Comment:
You don't need `--recursive` here. You're not copying a directory.

Actually you don't need to copy the requirements at all.